packages/development-proxy/mixin.core.js

```diff
@@ -20,29 +31,32 @@ class ProxyMixin extends Mixin {
               req.headers.accept && !req.headers.accept.includes('text/html')
             );
           },
-          {
-            target: proxyConfig,
-            changeOrigin: true,
-            logLevel: 'warn',
-          }
+          Object.assign(
+            {
+              target: proxyConfig,
+              changeOrigin: true,
+              logLevel: 'warn',
+            },
+            hooks
+          )
         )
       );
     }
 
     if (typeof proxyConfig === 'object') {
       debug('Using proxy object version', proxyConfig);
 
-      Object.entries(proxyConfig).forEach(([path, config]) => {
+      Object.entries(proxyConfig).forEach(([path, { target }]) => {
```

Comment:
I this the only option that we want to support in the config?

Reply:
Not sure, but we should not give the impression that someone could use the functions here.

We can always add more later, too.

What would your suggestion be?